CHANGES.md

```diff
@@ -36,6 +36,8 @@ To be released.
         Instead, a newly added static class `StoreExtension` provides
         an extension method of the same name.
      -  Added `IStore.IterateStateReferences()` method.
+ -  `Swarm` constructor became to receive `BlockChain<T>` and have a type
+    parameter `T`.  [[#324]]
```

Comment:
Actually, `Swarm`(not constructor, but class) became to have a type parameter. how about write them separately?

Reply:
I separated the changelog.